lib/contexts/CharactersContext/CharactersContext.tsx

```diff
@@ -197,12 +197,14 @@ const defaultCondensedCharacter: ICharacter = {
     { name: "Severe", value: "" },
   ],
   refresh: 3,
+  notes: "",
```

Comment:
must: because notes didn't exist before lets leave the default value to `undefined` otherwise we would need to add code to the character migration logic and I don't think it would be worth it.